app/policies/vehicle_report_policy.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class VehicleReportPolicy < ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    user_in_access_group?
+    # true
+  end
+
+  # def show?
```

Comment:
We're not using these comments. Let's get rid of them

Reply:
comments removed